packages/domainproperties/src/components/assay/AssayPropertiesPanel.tsx

```diff
@@ -170,51 +252,119 @@ export class AssayPropertiesPanel extends React.PureComponent<Props, State> {
                         <Col xs={12}>{children}</Col>
                     </Row>
                 }
-                {this.renderBasicProperties()}
-                {!basePropertiesOnly && this.renderImportSettings()}
-                {this.renderEditSettings()}
+                <Col className='domain-field-no-padding-right' xs={11} lg={5}>
+                    {this.renderBasicProperties()}
+                    {this.renderEditSettings()}
+                </Col>
+                <Col className='domain-field-no-side-padding' xs={1}/>
+                <Col xs={11} lg={6}>
+                    {!basePropertiesOnly && this.renderImportSettings()}
+                </Col>
             </Form>
         )
     }
 
+    getHeaderClass = () => {
+        const { panelStatus } = this.props;
+        const { collapsed, validProperties } = this.state;
+        let classes = 'domain-panel-status-icon';
+
+        if (collapsed) {
+            if (validProperties && panelStatus === 'COMPLETE') {
+                return classes + ' domain-panel-status-icon-green';
+            }
+            return (classes + ' domain-panel-status-icon-blue');
+        }
+
+        return classes;
+    };
+
+    getHeaderIcon = () => {
+        const { panelStatus } = this.props;
+        const { validProperties } = this.state;
+
+        if (!validProperties || panelStatus === 'TODO') {
+            return faExclamationCircle;
+        }
+
+        return faCheckCircle;
+    };
+
+    getHeaderIconComponent = () => {
+
+        return (
+            <span className={this.getHeaderClass()}>
+                <FontAwesomeIcon icon={this.getHeaderIcon()}/>
+            </span>
+        )
+    };
+
+    getHeaderIconHelpMsg = () => {
+        const { panelStatus } = this.props;
+        const { validProperties } = this.state;
+
+        if (!validProperties) {
+            return "This section has errors."
+        }
+
+        if (panelStatus === 'TODO') {
+            return "This section does not contain any user defined fields.  You may want to review."
+        }
+
+        return undefined;
+    };
```

Comment:
Similar comment as above for these functions (getHeaderIcon, getHeaderIconComponent, getHeaderIconHelpMsg), any way to share these between DomainForm and AssayPropertiesPanel?